v1/api/swagger/paths/hosts@{hostId}@reimage.yaml

```
@@ -5,6 +5,7 @@ post:
   summary: Reimage Host by ID
   description: >-
     Re-deploys a host to the same machine.
+    Warning, all drives will be erased!
```

It's unfortunate the lines get munged into a single line in the generated files below. Maybe capitalize it to make it more clear.
```suggestion
    WARNING -- all drives will be erased!
```